scaling_performance/host_practices.adoc

```diff
@@ -57,42 +57,94 @@ include writing a large amount of log data.
 etcd is a distributed key-value store that {product-title} uses for
 configuration.
 
+Version 3 of etcd brings with it important scalability and performance improvements that reduce CPU, memory, network and disk requirements for any size cluster.  OpenShift 3.5 introduced etcd version 3.1.
```

Comment:
Last sentence dropped.  And I've added a table so it makes the sentence redundant anyway.